### PR TITLE
Enable MacStadium VMs for KBFS

### PR DESCRIPTION
**Pull request by jzila**

### CODE REVIEW

File: Jenkinsfile [hunk 1]
```diff
@@ -94,112 +93,117 @@ helpers.rootLinuxNode(env, {
...
-                        parallel (
-                            pull_mysql: {
-                                if (startKbweb) {
-                                    mysqlImage.pull()
-                                }
-                            },
-                            pull_gregor: {
-                                if (startKbweb) {
-                                    gregorImage.pull()
+                    //test_windows: {
```
Comment (jzila) [inline Jenkinsfile]: We still intend on moving them to Jenkins; just haven't ironed out the bugs yet.

File: Jenkinsfile [hunk 2]
```diff
@@ -94,112 +93,117 @@ helpers.rootLinuxNode(env, {
...
+                                        },
+                                        pull_gregor: {
+                                            gregorImage.pull()
+                                        },
+                                        pull_kbweb: {
+                                            kbwebImage.pull()
+                                        },
+                                    )
+                                    retry(5) {
+                                        sh "docker-compose up -d mysql.local"
```
Comment (jzila) [inline Jenkinsfile]: Docker bugs. Sometimes it fails.

File: Jenkinsfile (hunk 1 above)
Comment (songgao) [inline Jenkinsfile]: got you thanks!

File: Jenkinsfile (hunk 2 above)
Comment (songgao) [inline Jenkinsfile]: :'(

**Pull request merged**